ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -603,7 +603,7 @@ open class DrawerController: UIViewController {
     private func updatePreferredContentSize(isExpanded: Bool) {
         isPreferredContentSizeBeingChangedInternally = true
         if isExpanded {
-            let screenHeight: CGFloat = UIScreen.main.bounds.height
+            let screenHeight: CGFloat = view.window?.screen.bounds.height ?? 0
```

Comment:
Should there be a `guard let window = view.window else {` at the top of this function instead? Not sure we really want to go through all this if there isn't a window.

Reply:
i thought about this. but then, we shouldn't be really updating any contentsize if we aren't part of the window hierarchy. there isn't any preferredcontentsize to update.

Reply:
addressed in next commit.